### DISCUSSION TITLE
[QUESTION] how to change default disk in longhorn

### DISCUSSION THREAD

**Discussion by btdan** [longhorn/longhorn#4631] [Q&A]
Hello,
## Question
How to locate the local disk in the yaml file?
I want to locathe the local disk such as /data/longhorn to longhorn.
And I have already known how to modify using the longhorn UI. However, if there are more than 20 node to modify , it is not ideal to modify it by hand.
I want to know how to modify https://raw.githubusercontent.com/longhorn/longhorn/v1.3.1/deploy/longhorn.yaml to solve this problem.
I have try to find the /var/lib/longhorn/  and replace with /data/longhorn which exist on the local disk. In detail, I modify two place in the longhorn.yaml file, just as follows:
 - name: longhorn
          mountPath: /var/lib/longhorn/
          mountPropagation: Bidirectional

and 
       - name: longhorn
        hostPath:
          path: /var/lib/longhorn/

However , it does not work.
Is there any method to change the disk in longhorn with the help of longhorn.yaml ?
Thanks very much.
## Environment

 - Longhorn version: 1.3.1
 - Kubernetes version: 1.2.4

**Comment by c3y1huang** [comment]
Ref https://longhorn.io/docs/1.3.1/best-practices/#configuring-default-disks-before-and-after-installation

**Comment by btdan** [comment]
thanks  very much
